### DISCUSSION TITLE
Front-end crash when editing first message

### DISCUSSION THREAD

**Discussion by leobenkel** [open-webui/open-webui#3320] [General]
# Bug Report

## Description

**Bug Summary:**

The chat breaks when the first message is edited

**Steps to Reproduce:**

* Drag a document ( i havent tried without the document )
* Ask a question
* Wait for answer
* Edit the very first message in this chat
* Errors in the console

**Expected Behavior:**

The chat should restart with this new initial message

**Actual Behavior:**

The chat is broken and can't be used anymore

## Environment

all latest. 
Firefox

## Reproduction Details

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the browser console logs.

## Logs and Screenshots

**Browser Console Logs:**
```
Uncaught (in promise) TypeError: currentMessage is undefined
    update Chat.svelte:109
    update scheduler.js:115
    flush scheduler.js:79

Uncaught (in promise) TypeError: history.messages[parentId] is undefined
    sendPrompt Chat.svelte:427
    confirmEditMessage Messages.svelte:82
```

**Comment by leobenkel** [comment]
I have just tested to edit the first message without uploading a document first and this crash as well.

**Comment by tjbck** [comment]
Can't reproduce on my machines, keep us updated if you find out the solution for the issue!

**Reply by leobenkel** [reply]
you are able to edit the first message, resubmit and it works fine ?
